src/api/server/authn.rst

```diff
@@ -345,3 +345,124 @@ headers to CouchDB with related requests:
 
 Note that you don't need to request :ref:`session <api/auth/session>`
 to be authenticated by this method if all required HTTP headers are provided.
+
+.. _api/auth/jwt:
+
+JWT Authentication
+====================
+
+.. note::
+    To use this authentication method make sure that the
+    ``{chttpd_auth, jwt_authentication_handler}`` value is added to the
+    list of the active :config:option:`chttpd/authentication_handlers`:
+
+    .. code-block:: ini
+
+        [chttpd]
+        authentication_handlers = {chttpd_auth, cookie_authentication_handler}, {chttpd_auth, jwt_authentication_handler}, {chttpd_auth, default_authentication_handler}
+
+`JWT authentication` enables CouchDB to use externally generated JWT tokens
```

Comment:
`JWT authentication` needs different formatting, either double-backticks, or perhaps you meant bold/otherwise?

If you meant it to be a [phrase reference](https://docutils.sourceforge.io/docs/user/rst/quickref.html#hyperlink-targets) then it's incomplete.

Reply:
I will change this but I note single backticks is used everywhere else earlier in this file, including the 'proxy authentication' section I used as my base.

Reply:
OK, I'll file another PR to fix all of that. Thanks for letting me know.